a simple 'int main(){}' function won't generate references to SSP functions
when using -fstack-protector which means systems that dont have SSP support
wont get properly detected as lacking support.  instead, create a big buffer
on the stack and use it as that seems to do the trick.

https://bugzilla.mindrot.org/show_bug.cgi?id=1538
https://bugs.gentoo.org/244776

--- openssh-5.1p1/configure.ac
+++ openssh-5.1p1/configure.ac
@@ -145,8 +145,8 @@ int main(void){return 0;}
 		      AC_MSG_CHECKING(if $t works)
 		      AC_RUN_IFELSE(
 			[AC_LANG_SOURCE([
-#include <stdlib.h>
-int main(void){exit(0);}
+#include <stdio.h>
+int main(void){char foo[[1024]];return sprintf(foo, "moo cow") == 7;}
 			])],
 			[ AC_MSG_RESULT(yes)
 			  break ],
